stable/datadog/OWNERS

```diff
@@ -2,7 +2,9 @@ approvers:
 - hkaj
 - irabinovitch
 - xvello
+- charlyf
```

Comment:
Yes please! You'll need to open an issue to become a trusted collaborator first though.